source/adapters/level_zero/common.hpp

```diff
@@ -533,7 +533,8 @@ extern thread_local int32_t ErrorAdapterNativeCode;
                                       ur_result_t ErrorCode,
                                       int32_t AdapterErrorCode);
 
-#define L0_DRIVER_INORDER_MIN_VERSION 29534
+#define L0_DRIVER_INORDER_MIN_MINOR_VERSION 6
```

Comment:
Why change the name of the macro? Could this break some existing code? Also `MIN_MINOR` seems strange. If a renaming is necessary, why not just use `L0_DRIVER_INORDER_MINOR_VERSION` and `L0_DRIVER_INORDER_PATCH_VERSION`.

Reply:
Updated in the latest version to L0_DRIVER_INORDER_MINOR_VERSION and L0_DRIVER_INORDER_PATCH_VERSION since I want this to be explicit. We need to be able to target a driver's minor and patch version.

Reply:
Can we just move these values to inside `allowDriverInOrderLists()` so that the definition is next to the usage, or is it intended that these in-order driver version values could be used more generally.

Reply:
I had thought it should be generic, but @EwanC you are right, these defines should be limited to the allowDriverInOrderLists() now that the code has been merged, let me make that change.